src/engine/BoundManager.cpp

```diff
@@ -27,7 +27,10 @@ BoundManager::BoundManager( Context &context )
     : _context( context )
     , _size( 0 )
     , _tableau( nullptr )
+    , _consistentBounds( &_context )
+    , _firstInconsistentTightening( 0, 0.0, Tightening::LB )
 {
+  _consistentBounds = true;
```

Comment:
Use 4-space instead for 2-space for indentation to be consistent with rest of the code base?

Reply:
Done.